kube-controllers/pkg/controllers/node/ipam.go

```diff
@@ -1074,6 +1170,140 @@ func (c *ipamController) kubernetesNodeForCalico(cnode string) (string, error) {
 	return getK8sNodeName(*calicoNode)
 }
 
+func (c *ipamController) incrementReclamationMetric(block string, node string) {
+	pool := c.poolManager.poolsByBlock[block]
+	metricNode := node
```

Comment:
You shouldn't need to redeclare this variable - you can just use the `node`  string that's passed in. The string is passed by value so we don't need to worry about modifying it.